app/assets/stylesheets/newflow_layout.scss

```diff
@@ -18,82 +44,346 @@ body {
 #signinup-form {
     max-width: 75rem;
     border-top: 0;
-    margin-top: 3rem;
     margin-left: auto;
     margin-right: auto;
+    margin-bottom: 3rem;
+    margin-top: 4rem;
+    padding-left: 14px;
+    padding-right: 14px;
+
+    .step-counter {
+        height: 30px;
+
+        > div {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            width: 94px;
+            height: 100%;
+
+            background-color: #f5f5f5;
+            border-radius: 1.5rem;
+        }
+
+        span {
+            font-size: 1.4rem;
+            font-weight: bold;
+            letter-spacing: -0.56px;
+            // padding: 1rem;
```

Comment:
Is this possibly going to be reinstated, or should it be removed?

Reply:
Removed it.